tools/fuzz/fuzz_redaction.py

```
@@ -0,0 +1,86 @@
+
+    stripped = name.strip()
+    # 不变式 2：空白姓名原样返回
+    if not stripped:
+        assert result == text, "空白姓名不得改写文本"
+        return
+
+    # 不变式 3：CJK 姓名连续出现必须被脱敏（CJK 无词边界，连续即命中）
+    if _is_pure_cjk(stripped) and stripped in text:
+        assert stripped not in result, f"漏脱敏：{stripped!r} 仍连续出现在输出"
+
+    # 不变式 4：宽松意义（去空白+小写）下不存在 → 不误伤
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)